spring-cloud-contract-verifier/pom.xml

```diff
@@ -213,6 +213,24 @@
 			<artifactId>xercesImpl</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.jetbrains.kotlin</groupId>
+			<artifactId>kotlin-compiler-embeddable</artifactId>
+			<version>${kotlin.version}</version>
```

Comment:
If https://github.com/spring-cloud/spring-cloud-contract/pull/1151 is merged before this (which I think will be the case) then this will be fixed while merging master into this branch.

Reply:
Fixed